clients/src/main/java/org/apache/kafka/common/security/JaasUtils.java

```diff
@@ -44,7 +44,11 @@
      * @return JAAS configuration object
      */
     public static String jaasConfig(String loginContextName, String key) throws IOException {
-        AppConfigurationEntry[] configurationEntries = Configuration.getConfiguration().getAppConfigurationEntry(loginContextName);
+        return jaasConfig(Configuration.getConfiguration(), loginContextName, key);
+    }
+
+    public static String jaasConfig(Configuration jaasConfig, String loginContextName, String key) throws IOException {
```

Comment:
Renamed to `jaasConfigOption` since the underlying field from which the value is returned is called `options`.